### PR TITLE
Publish git HEAD tag when publishing docker image

### PR DESCRIPTION
**Pull request by simao**
Do not publish latest

Bump libats and libtuf

Remove influxdb

### CODE REVIEW

File: src/main/scala/com/advancedtelematic/director/daemon/DaemonBoot.scala [hunk 1]
```diff
@@ -31,8 +31,8 @@ object DaemonBoot extends BootApp
     with MetricsSupport
     with DatabaseMetrics
     with MessageListenerSupport
-    with InfluxdbMetricsReporterSupport
     with AkkaHttpRequestMetrics
+    with CheckMigrations
```
Comment (houcros) [inline src/main/scala/com/advancedtelematic/director/daemon/DaemonBoot.scala]: Shouldn't we `CheckMigrations` at `Boot` as well? We do it for campaigner and device-registry.

Comment (simao) [reply src/main/scala/com/advancedtelematic/director/daemon/DaemonBoot.scala]: Yes! Thanks

**Comment by simao** [comment]
please merge when needed

**Comment by koshelev** [comment]
@simao removing you from the team for the next two weeks. Have a nice vacation ;)

**Pull request merged**